v5/getting-started/macos.rst

```diff
@@ -2,49 +2,88 @@
 Installing on macOS
 ===================
 
-There are currently two ways to install PROS 3 on macOS. The recommended method uses `Homebrew <https://brew.sh/>`_, and the other method involves installing components manually.
-
-Recommended Method (Homebrew)
------------------------------
-
-The recommended method of installing PROS 3 for macOS involves using `Homebrew <https://brew.sh/>`_.
-
-1. If you do not already have Homebrew installed, install it by following the instructions on `their site <https://brew.sh>`_. This will take a while, and may prompt you to follow some additional instructions.
-2. Once you have Homebrew installed, run :code:`brew tap osx-cross/arm && brew install arm-gcc-bin` to register a repository with Homebrew that contains the toolchain used to build PROS projects, and then install the toolchain.
-3. Run :code:`brew tap purduesigbots/pros` to register the PROS Homebrew repository with Homebrew.
-4. Run :code:`brew install --cask pros-editor` to install the PROS Editor (the CLI will also be installed). This may also take a while.
-5. (Optional) If you are planning to use the Vision Sensor, you will likely need to also install the VEX Vision Utility to set up signatures. Run :code:`brew install vcs-vision` to do this. Once installed, you'll be able to run the program by looking for "vcs_vision" in Spotlight.
+.. note::
+   The recommended method for installing PROS on all platforms is now using our
+   VS Code extension. Instructions for this can be found on the :doc:`./index`
+   page.
+   
+   If you do not want to use VS Code, the method below will allow you to install
+   the old, deprecated fork of Atom we call the PROS Editor or the CLI by iteslf.
+
+   If you are an advanced user looking to install the PROS CLI through Python pip,
+   check out the `Installing the CLI <#install-the-cli>`__ section below.
+
+Homebrew method
+---------------
+
+1. If you do not already have Homebrew installed, install it by following the
+   instructions on `their site <https://brew.sh>`_. This will take a while, and
+   may prompt you to follow some additional instructions.
+2. Once you have Homebrew installed, run :code:`brew tap osx-cross/arm && brew install arm-gcc-bin`
+   to register a repository with Homebrew that contains the toolchain used to
+   build PROS projects, and then install the toolchain.
+3. Run :code:`brew tap purduesigbots/pros` to register the PROS Homebrew
+   repository with Homebrew.
+4. Run :code:`brew install --cask pros-editor` to install the PROS Editor
+   (the CLI will also be installed). This may also take a while.
+5. (Optional) If you are planning to use the Vision Sensor, you will likely need
+   to also install the VEX Vision Utility to set up signatures. Run
+   :code:`brew install vcs-vision` to do this. Once installed, you'll be able to
+   run the program by looking for "vcs_vision" in Spotlight.
 6. That's it! You can now start using PROS 3.

```

Comment:
```suggestion
6. That's it! You can now start using PROS for V5.
```

We should make this non version specific, for PROS 4 and whatever comes next.

Reply:
yeah good point, this wording is really old. tbh it doesnt even need to say "for v5" either